changelog.json

```diff
@@ -1,5 +1,30 @@
 {
   "releases": {
+    "2.4": [
+      "[New] Set Git proxy environment variables from system configuration automatically - #9154",
+      "[Added] Enable creating a GitHub issue from app menu - #8989",
+      "[Added] Add editor support for IntelliJ Idea on macOS - #8398. Thanks @gssbzn!",
+      "[Added] Add editor support for PhpStorm and update WebStorm - #7195. Thanks @dacoto97!",
+      "[Added] Add shell support for Windows Terminal - #7854. Thanks @AndreiMaga!",
+      "[Added] Support keyboard scrolling in diffs - #9074",
+      "[Fixed] Checkout correct PR branch from “open this in GitHub Desktop” link on GitHub.com pull request page - #9108",
+      "[Fixed] Ensure branch protection warning works when branch names contain special characters - #9086",
+      "[Fixed] Ensure intro page is responsive when resizing the window - #9175",
+      "[Fixed] Ensure local branch is also deleted when deleting a branch that no longer exists on the remote - #8831",
+      "[Fixed] Fix graphic indicating files selected in Dark Mode - #8716. Thanks @say25!",
+      "[Fixed] Support selecting text in error dialogs - #7606",
+      "[Fixed] Preserve the footer within a resizable list when shrinking or expanding - #9344",
+      "[Fixed] Prevent bringing changes to another branch from failing silently - #9038",
+      "[Fixed] Save the user's position within \"History\" and \"Changes\" tabs when switching - #9061",
+      "[Improved] Update the remote url when a repository's name changes on GitHub - #8590",
+      "[Improved] Allow \"Open in Desktop\" button from pull requests on GitHub to open in a local fork in Desktop - #9012",
+      "[Improved] Direct new issues created from a local fork to its upstream on GitHub - #9232",
```

Comment:
I'm not sure this one should be a release note since it's just our expected behavior as part of the new issue creation and doesn't represent a change from the existing prod release.
```suggestion
```